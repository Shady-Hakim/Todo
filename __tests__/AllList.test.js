/**
 * @format
 */

import 'react-native';
import React from 'react';
import AllList from '../src/screens/allList/allList';
import renderer from 'react-test-renderer';
jest.useFakeTimers();

test('item render', () => {
    const component = renderer.create(<AllList />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});