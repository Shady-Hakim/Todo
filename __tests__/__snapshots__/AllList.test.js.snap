// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`item render 1`] = `
<View>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#5dcff3",
        "borderRadius": 10,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-between",
        "marginVertical": 5,
        "opacity": 1,
        "paddingHorizontal": 20,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            null,
          ]
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            null,
          ]
        }
      />
    </View>
    <Text
      style={
        Array [
          Object {
            "color": "#fff",
          },
          null,
        ]
      }
    />
  </View>
</View>
`;
